git/compat.py

```diff
@@ -56,7 +56,9 @@ def safe_decode(s):
         return s
     elif isinstance(s, six.binary_type):
         if PRE_PY27:
-            return s.decode(defenc)  # we're screwed
+            # Python 2.6 does not support the `errors` argument, so we cannot
+            # control the replacement of unsafe chars in it.
+            return s.decode(defenc)
```

Comment:
BTW: I was about to merge a few PRs, but work is ...catching up earlier than expected. Will try again tonight, and if these delays become unbearable to you, I'd love to make you a collaborator. Just let me know what you think.
Cheers :) !